package.json

```diff
@@ -17,6 +17,13 @@
       "pre-push": "yarn test"
     }
   },
+  "jest":{
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$":
```

Comment:
Is this a canonical list copied from somewhere?

Reply:
Yes, from Jest documentation: https://facebook.github.io/jest/docs/en/webpack.html#mocking-css-modules